main.go

```diff
@@ -137,6 +138,15 @@ func generate(request pluginpb.CodeGeneratorRequest) (*pluginpb.CodeGeneratorRes
 			}
 		} else if k == "additional_empty_schema" {
 			messagesWithEmptySchema = strings.Split(v, "+")
+		} else if k == "disable_validation" {
```

Comment:
disable_validation is confusing as we still add some constraints regardless. Let's change it to  `disable_kube_markers and disableKubeMarkers` instead of `disable validation`